client/src/main/java/com/influxdb/client/BucketsQuery.java

```diff
@@ -0,0 +1,112 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class BucketsQuery {
+
+    /**
+     * offset  (optional)
+     */
+    @Nullable
+    private Integer offset;
+
+    /**
+     *  (optional, default to 20)
```

Comment:
Fix `JavaDoc` formatting:
```suggestion
     * Limit. (optional, default to 20)
```